### ISSUE TITLE
NTR, pulmo: Lipid pneumonia

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6396]
**Preferred term label:**
Lipid pneumonia

**Synonyms**
Cholesterol Pneumonia

**Definition (free text, please give PubMed ID)**
Specific form of lung inflammation that develops when lipids enter the bronchial tree

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by GrieseAG** [comment]
child terms: 
exogenous LP: Lipid comes from the outside, for example by inhaling aromatic oil

endogenous LP: Lipid occurs from the body itself.  when an airway is obstructed, it is often the case that distal to the obstruction, lipid-laden macrophages and giant cells fill the lumen of the disconnected airspace

**Comment by pnrobinson** [comment]
It seems these are diseases that also have characteristic phenotypic features that distinguish them from other forms of pneumonia
https://www.ajronline.org/doi/10.2214/AJR.09.3040
e.g., 
Characteristically, chronic exogenous lipoid pneumonia manifests as an adipose-containing mass [1] (Fig. 8). Although the mass is typically irregular or spiculated as a result of chronic inflammation and secondary fibrosis, the presence of fat in the mass is, with a few exceptions, a diagnostic feature of exogenous lipoid pneumonia.


Endogenous lipoid pneumonia is a histopathologic diagnosis made on the basis of the characteristic appearance of the obstructive pneumonitis. Macroscopically, there is parenchymal consolidation that has a characteristic yellowish discoloration due to the accumulation of lipid in the alveoli [18]. Histologically, there is an accumulation of lipid-filled macrophages and eosinophilic proteinaceous material derived from degenerating cells, including surfactant from type II pneumocytes, in the alveoli distal to the bronchial obstruction.

**Comment by pnrobinson** [comment]
Added terms. Note that it is "lipoid" (with o) Pneumonia.
